### ISSUE TITLE
Decrease margin around icons

### ISSUE THREAD

**Issue by kajetanschuler** [torryharris/Skycons#8]
Hi guys,

awesome work!

Is there a way to decrease the margin around the icons?

Greets

**Comment by ShanmugapriyaVasu** [comment]
No, Its not possible to decrease the margin of  these views. Will be taken up for the future enhancement.
Thank you.
